.github/meta.yaml

```diff
@@ -37,7 +37,7 @@ outputs:
         - requirements-parser >=0.2.0
         - shap >=0.40.0
         - texttable >=1.6.2
-        - woodwork <=0.19.0
+        - woodwork >=0.21.0
```

Comment:
Are the changes in this PR incompatible with older versions of woodwork?

Reply:
Confirmed that `0.19.0` works fine while `0.20.0` has dependency issues. I updated the dependencies appropriately!